Assets/MixedRealityToolkit.Examples/Demos/EyeTracking/General/Scripts/FollowEyeGaze.cs

```diff
@@ -23,6 +23,11 @@ private void Update()
             var eyeGazeProvider = CoreServices.InputSystem?.EyeGazeProvider;
             if (eyeGazeProvider != null)
             {
+                if(!eyeGazeProvider.UseEyeTracking)
+                {
+                    eyeGazeProvider.UseEyeTracking = true;
```

Comment:
Why is this set to true in the code? And why in the Update function? Maybe you want to switch between different head and eye gaze modes at run time?

Reply:
The scene uses the script LoadProfilesOnStartup.cs , this script resets the profile on start. 

Calling the profile initialization on runtime is instantiating a new GazeProvider.cs on the Camera object, and even if we configure GazeProvider to have UseEyeTracking = true on the parent scene, the new GazeProvider has the default value of UseEyeTracking = false.  

This is the reason having UseEyeTracking = true as default was apparently solving the EyeTracking Demo scenes issues.

Solutions for this issue include not changing profile on startup, changing UseEyeTracking on code, or implementing a better way of Profile change which is probably on the way.

Reply:
@vaoliva  instead of forcefully activating that flag - maybe we just want to take advantage of your patched up util function for getting the eye tracking data (InputRayUtils) and use that for determining the gameobject position - that way you don't have to interfer with what the user has set up

Reply:
The eye tracking scenes are actually using focus info. So if gaze is not eye based scenes would be broken. Also [documentation](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/EyeTracking/EyeTracking_BasicSetup.html#enabling-eye-tracking-in-the-gazeprovider) suggests user can set up UseEyeTracking on code.

Reply:
ah, i didn't see they're using the focus target as well - but then we should probably just not set that flag at all here - and just refer to the documentation - we could prompt a warning telling the user that this flag isnt' enabled but not set it ourselves. We're not doing that in any other example, so wouldn't do it in just one of our scripts.